<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Level</class>
 <widget class="QWidget" name="Level">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Hint</string>
   </property>
  </widget>
  <widget class="QLabel" name="levelNumberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dummyAnswer3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>236</y>
     <width>159</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">answerGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="correctAnswer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>137</y>
     <width>159</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">answerGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="dummyAnswer2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>203</y>
     <width>159</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">answerGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="dummyAnswer1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>159</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">answerGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="answerGroup"/>
 </buttongroups>
</ui>
